vllm/model_executor/models/registry.py

```diff
@@ -201,6 +201,10 @@
     **_FALLBACK_MODEL,
 }
 
+_SUBPROCESS_COMMAND = [
```

Comment:
this is needed since we would like to override the subprocess command.

We may leverage par format to packing things together, so may need to customize subprocess.run parameters.

Reply:
maybe we should put some comments here.

Reply:
Added comments